docker-compose.yml

```diff
@@ -170,16 +170,17 @@ services:
       args:
         BASE_IMAGE: 'sentry-onpremise-local'
     command: '"0 0 * * * gosu sentry sentry cleanup --days $SENTRY_EVENT_RETENTION_DAYS"'
-  nginx:
+  frontend:
     << : *restart_policy
     ports:
       - '9000:80/tcp'
-    image: "nginx:1.16"
-    volumes:
-      - type: bind
-        read_only: true
-        source: ./nginx
-        target: /etc/nginx
+    image: 'us.gcr.io/sentryio/frontend:latest'
+    environment:
```

Comment:
There's also an additional variable which might be nice to add:

```
LOG_FORMAT: json
```

And this will dump out structured access logs. It's not doing this today, so didn't set it.